test/test_models.py

```diff
@@ -42,7 +51,7 @@ def get_available_video_models():
 ]
 
 
-class Tester(unittest.TestCase):
+class Tester(TestCase):
```

Comment:
What do you think if we now change the name of the `Tester` to be something like `ModelTester`, so that the expect files have a common prefix?